.github/workflows/backport.yml

```diff
@@ -0,0 +1,25 @@
+name: Backport
+on:
+  pull_request_target:
+    types:
+      - closed
+      - labeled
```

Comment:
I guess this should only be closed then? If we only want to run on merged PRs?

Reply:
It's taken from step 1 in https://github.com/tibdex/backport#usage directly

Reply:
I mean, I haven't tested this, but it feels a bit weird. It should only run whenever a pull request gets closed, see:
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target

Reply:
Now that I think about it, it's probably so you can backport after merging by adding labels. (Which would have avoided the work in #7775 )

Reply:
Makes sense!